xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-default/src/main/java/org/xwiki/notifications/internal/DefaultModelBridge.java

```diff
@@ -52,8 +54,6 @@
 @Singleton
 public class DefaultModelBridge implements ModelBridge
 {
-    private static final String EVENT_TYPE_FIELD = "eventType";
-
     private static final SpaceReference NOTIFICATION_CODE_SPACE = new SpaceReference("Code",
         new SpaceReference("Notifications",
             new SpaceReference("XWiki", new WikiReference("toChange"))
```

Comment:
> new WikiReference("toChange")

seriously... (yes I know, not your fault)

Would be much cleaner to use LocalDocumentReference instead.